app/policies/plan_policy.rb

```diff
@@ -28,17 +28,25 @@ def export?
     @plan.readable_by?(@user.id)
   end
 
-  def show_export?
-    @plan.readable_by?(@user.id)
-  end
+# TODO: There is no route to this function on the controller
+#  def show_export?
```

Comment:
Uncomment this, otherwise the export page will not be displayed. See the route here https://github.com/DigitalCurationCentre/roadmap/blob/development/config/routes.rb#L201